annotation-client/src/test/java/eu/europeana/annotation/client/integration/webanno/tag/BodyTypesTest.java

```diff
@@ -0,0 +1,354 @@
+import eu.europeana.annotation.definitions.model.body.TagBody;
+import eu.europeana.annotation.definitions.model.body.impl.EdmPlaceBody;
+import eu.europeana.annotation.definitions.model.body.impl.FullTextResourceBody;
+import eu.europeana.annotation.definitions.model.body.impl.SemanticTagBody;
+import eu.europeana.annotation.definitions.model.body.impl.TextBody;
+import eu.europeana.annotation.definitions.model.entity.impl.EdmPlace;
+import eu.europeana.annotation.definitions.model.graph.Graph;
+import eu.europeana.annotation.definitions.model.search.SearchProfiles;
+import eu.europeana.annotation.definitions.model.search.result.AnnotationPage;
+import eu.europeana.annotation.definitions.model.target.Target;
+import eu.europeana.annotation.definitions.model.vocabulary.BodyInternalTypes;
+import eu.europeana.annotation.definitions.model.vocabulary.MotivationTypes;
+
+public class BodyTypesTest extends BaseTaggingTest {
+
+	static final String VALUE_TESTSET = "generator_uri: \"http://test.europeana.org/45e86248-1218-41fc-9643-689d30dbe651\"";
+	static final String VALUE_ID = "anno_id:";
+	static final String VALUE_TARGET = "target_uri:\"http://data.europeana.eu/item/09102/_UEDIN_214\"";
+	static final String VALUE_TARGET_TAG = "target_uri:\"http://data.europeana.eu/item/000002/_UEDIN_214\"";
+	static final String VALUE_TARGET_LINK = "target_uri:\"http://data.europeana.eu/item/123/xyz\"";
+	static final String VALUE_TARGET_LINK_SEMANTIC = "target_uri:\"http://data.europeana.eu/item/2059207/data_sounds_T471_5\"";
+	static final String VALUE_DESCRIBING_TARGET_SCOPE = "target_uri:\"http://data.europeana.eu/item/07931/diglit_uah_m1\"";
+	static final String VALUE_BODY_URI = "body_uri:\"http://www.geonames.org/2988507\"";
+	static final String VALUE_DESCRIBING_BODY_VALUE = "body_value:\"... this is the textual description of the item ...\"";
+	static final String VALUE_BODY_VALUE = "body_value:\"trombone\"";
+	static final String VALUE_BODY_LINK_RELATION = "link_relation:\"isSimilarTo\"";
+	static final String VALUE_BODY_LINK_RESOURCE_URI = "link_resource_uri:\"http://thesession.org/tunes/52\"";
+	static final String VALUE_BODY_SPECIFIC_RESOURCE = "body_uri:\"http://www.geonames.org/2988507\""; // source
+	static final String VALUE_BODY_FULL_TEXT_RESOURCE = "body_value:\"... complete transcribed text in HTML ...\"";
+
+	static final int TOTAL_BY_ID_FOUND = 1;
+	
+	protected Logger log = LogManager.getLogger(getClass());
+	
+	private AnnotationSearchApiImpl annotationSearchApi;
+	
+	
+	/**
+	 * Create annotations data set before each test execution
+	 * 
+	 * @throws IOException
+	 */
+	@BeforeEach
+	public void createAnnotationSearchApi() throws JsonParseException, IOException {
+		annotationSearchApi = new AnnotationSearchApiImpl();
+	}
+	
+	@Test
+	public void createSemanticTag() throws IOException, JsonParseException, IllegalAccessException, IllegalArgumentException, InvocationTargetException {
```

Comment:
this is a search not a create test, please rename

Reply:
changed to search...